### PR TITLE
Remove swagger dependencies

### PR DESCRIPTION
**Pull request by msbw2**
Remove swagger dependencies.

### CODE REVIEW

**Review by jmprieur — APPROVED** [review]
Lgtm.
Thanks @msbw2

File: tests/DevApps/ciam/myWebApi/Program.cs [hunk 1]
```diff
@@ -11,19 +11,11 @@
     .AddMicrosoftIdentityWebApi(builder.Configuration.GetSection("AzureAd"));
 
 builder.Services.AddControllers();
-// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+// Learn more about configuring OpenAPI at https://aka.ms/aspnetcore/swashbuckle
```
Comment (westin-m) [inline tests/DevApps/ciam/myWebApi/Program.cs:14]:
```suggestion
// Learn more about configuring OpenAPI at https://learn.microsoft.com/en-us/aspnet/core/fundamentals/openapi/aspnetcore-openapi?view=aspnetcore-8.0&tabs=visual-studio%2Cminimal-apis
```
**Sugestion:** The currently linked docs seem to be more Swagger-focused. I think this is a better resource, but I could not find an aka.ms link

**Review by kellyyangsong — APPROVED** [review]

**Review by jennyf19 — APPROVED** [review]
This is great, thanks @msbw2

**Pull request merged**